### PR TITLE
Fix error in Order detail page on VM3.4

### PR DESCRIPTION
**Pull request by dzungtran**
- Update latest library
- Fix error in Order detail page on VM3.4

### CODE REVIEW

File: src/paymentwall.php [hunk 1]
```diff
@@ -73,31 +75,41 @@ function plgVmOnStoreInstallPaymentPluginTable($jplugin_id)
...
+        // Prepare data that should be stored in the database
+        $dbValues['order_number'] = $order['details']['BT']->order_number;
+        $dbValues['payment_name'] = $this->_currentMethod->payment_name;
+        $dbValues['virtuemart_paymentmethod_id'] = $cart->virtuemart_paymentmethod_id;
+        $dbValues['cost_per_transaction'] = $this->_currentMethod->cost_per_transaction;
+        $dbValues['cost_percent_total'] = $this->_currentMethod->cost_percent_total;
+        $dbValues['payment_currency'] = $this->_currentMethod->payment_currency;
+        $dbValues['payment_order_total'] = $order['details']['BT']->order_total;
+        $dbValues['tax_id'] = $this->_currentMethod->tax_id;
+        $this->storePSPluginInternalData($dbValues);
```
Comment (sevastyanovio) [inline src/paymentwall.php]: move to prepareData() method

Comment (dzungtran) [inline src/paymentwall.php]: Updated

File: src/paymentwall.php [hunk 2]
```diff
@@ -40,12 +40,18 @@ function getVmPluginCreateTableSQL()
 
     function getTableSQLFields()
     {
-        $sqlFields = array(
-            'id' => 'int(1) UNSIGNED NOT NULL AUTO_INCREMENT',
+         $sqlFields = array(
+            'id' => 'int(11) UNSIGNED NOT NULL AUTO_INCREMENT',
+            'virtuemart_order_id' => 'int(1) UNSIGNED',
```
Comment (sevastyanovio) [inline src/paymentwall.php]: int(11) ?

Comment (dzungtran) [inline src/paymentwall.php]: Updated .

**Pull request merged**